Assignment7+8/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en-us">
+<head>
+    <title>Joke Page</title>
+    <link rel="stylesheet" href="css/site.css">
+    <link rel="icon" type="image/x-icon" href="favicon.ico">

```

Comment:
```suggestion
    <link rel="icon" type="image/x-icon" href="images/favicon.ico">
```
Put your icon in an images folder